gelato-web3-script/src/web3-functions/GearboxClaimAndBribe/schema.json

```diff
@@ -0,0 +1,11 @@
+{
+  "web3FunctionVersion": "1.0.0",
+  "runtime": "js-1.0",
+  "memory": 128,
+  "timeout": 30,
+  "userArgs": {
+    "multisigClaimAddress": "string",
+    "claimTokenAddress": "string",
```

Comment:
Maybe rename this to bribeTokenAddress or handleTokenAddress or just tokenAddress as this contract or something very similar to it will likely be used for others who are not claiming and just using bribAll or bribBoth with balances they transfer into the contract.